### PR TITLE
Use new InvokerService for doing wakeup calls at regular intervals.

### PR DESCRIPTION
**Pull request by baldersheim**
@geirst PR

### CODE REVIEW

**Comment by baldersheim** [comment]
Failing unit test is unrelated and fixed in#20300

**Review by geirst — APPROVED** [review]
👍

File: searchcore/src/vespa/searchcore/proton/server/shared_threading_service.cpp [hunk 1]
```diff
@@ -18,7 +19,9 @@ SharedThreadingService::SharedThreadingService(const SharedThreadingServiceConfi
     : _warmup(cfg.warmup_threads(), 128_Ki, proton_warmup_executor),
       _shared(std::make_shared<vespalib::BlockingThreadStackExecutor>(cfg.shared_threads(), 128_Ki,
                                                                       cfg.shared_task_limit(), proton_shared_executor)),
-      _field_writer()
+      _field_writer(),
+      _invokeService(5ms),
```
[the remark this answers is not in the document]
Comment (baldersheim) [reply searchcore/src/vespa/searchcore/proton/server/shared_threading_service.cpp:23]: The intention is to use cfg.fw_cfg.reactionTime().
Once this has passed a round of 2 with performance tests The sequenced executor will get a higher default and just use that while the wakeup service will use the configured frequency.

[the remark this answers is not in the document]
Comment (geirst) [reply searchcore/src/vespa/searchcore/proton/server/shared_threading_service.cpp:23]: 👍

**Pull request merged**